lib/components/src/blocks/ArgsTable/ArgValue.tsx

```diff
@@ -129,6 +138,12 @@ const ArgSummary: FC<ArgSummaryProps> = ({ value, initialExpandedArgs }) => {
   const summaryAsString = typeof summary.toString === 'function' ? summary.toString() : summary;
 
   if (detail == null) {
+    const cannotBeSafelySplitted = /[(){}[\]<>]/.test(summaryAsString);
+
+    if (cannotBeSafelySplitted) {
+      return <ArgText text={summaryAsString} simple={summaryAsString.includes('|')} />;
```

Comment:
Can you explain `simple` to me? Don't understand why we need it.

Reply:
@ajkl2533

Reply:
Sorry I have a vacation. Simple is turning on and off styling of values (border, background...). And I used it because when you have non splitted values it looked strange with styling.